### ISSUE TITLE
Building

### ISSUE THREAD

**Issue by voodookult** [xCENTx/Borderlands-GOTY-Enhanced-Trainer-Source#2]
Hi, how do I build the Trainer? I already have downloaded the code from GitHub and the trainer from your post.

**Comment by eoussama** [comment]
Any info on this one?

**Comment by voodookult** [comment]
yeah mate, sorry I didn't see it, I found a solution by myself but this video is much better, just do the same as he does https://www.youtube.com/watch?v=zSl1PgmhpwY&list=LL&index=1

if the .exe inside your bin\Release doesn't work, try the ones inside the folders bin\x64\Release (this one worked for me) or bin\x86\Release
